### PR TITLE
Fixes code cov build

### PR DESCRIPTION
**Pull request by dutterbutter**
**Summary of changes**
Changes introduced in this pull request:
- Fixes code cov build

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

**Review by austinabell — APPROVED** [review]
Why was the dep update necessary?

**Comment by dutterbutter** [comment]
> Why was the dep update necessary?

I thought it would resolve the security build issue from `tempdir `

**Comment by austinabell** [comment]
> > Why was the dep update necessary?
> 
> I thought it would resolve the security build issue from `tempdir `

Nah, that's just a warning, the issue is with multihash #839

File: node/forest_libp2p/src/service.rs [hunk 1]
```diff
@@ -317,13 +315,11 @@ where
...
                                 warn!("Failed to send a bitswap want_block: {}", e.to_string());
-                            } else {
-                                if let Some(chans) = self.bitswap_response_channels.get_mut(&cid) {
-                                    chans.push(response_channel);
+                            } else if let Some(chans) = self.bitswap_response_channels.get_mut(&cid) {
+                                chans.push(response_channel);
                                 } else {
-                                    self.bitswap_response_channels.insert(cid, vec![response_channel]);
+                                self.bitswap_response_channels.insert(cid, vec![response_channel]);
                                 }
```
Comment (ec2) [inline node/forest_libp2p/src/service.rs:322]: could you fix all the mucked up indentation here?

**Review by ec2 — APPROVED** [review]

File: node/forest_libp2p/src/service.rs (hunk 1 above)
Comment (austinabell) [reply node/forest_libp2p/src/service.rs:322]: 320 and 322 just need to be unindented

**Comment by vmx** [comment]
Thanks a lot @dutterbutter for this PR. I'm currently looking into adding coverate for rust-fil-proofs and this simplified version surely saved me a ton of time figuring out grcov myself.

**Pull request merged**